modules/widgets/contact-info.php

```diff
@@ -242,21 +242,50 @@ function form( $instance ) {
 			<p>
 				<label for="<?php echo esc_attr( $this->get_field_id( 'address' ) ); ?>"><?php esc_html_e( 'Address:', 'jetpack' ); ?></label>
 				<textarea class="widefat" id="<?php echo esc_attr( $this->get_field_id( 'address' ) ); ?>" name="<?php echo esc_attr( $this->get_field_name( 'address' ) ); ?>"><?php echo esc_textarea( $instance['address'] ); ?></textarea>
+			</p>
 
+			<p>
 				<input class="jp-contact-info-showmap" id="<?php echo esc_attr( $this->get_field_id( 'showmap' ) ); ?>" name="<?php echo esc_attr( $this->get_field_name( 'showmap' ) ); ?>" value="1" type="checkbox" <?php checked( $instance['showmap'], 1 ); ?> />
 				<label for="<?php echo esc_attr( $this->get_field_id( 'showmap' ) ); ?>"><?php esc_html_e( 'Show map', 'jetpack' ); ?></label>
 			</p>
 
+			<?php if ( has_filter( 'jetpack_google_maps_api_key' ) ) { ?>
+
+			<p class="jp-contact-info-admin-map" style="<?php echo $instance['showmap'] ? '' : 'display: none;'; ?>">
+				<?php
+					printf(
+						wp_kses(
+							__('API key <code>%1$s</code> is being set by <a href="%2$s" target="_blank" rel="noopener noreferrer">this</a> filter.', 'jetpack'),
```

Comment:
I'd recommend setting your IDE as recommended here:
https://github.com/Automattic/jetpack/blob/master/docs/development-environment.md#use-php-codesniffer-and-eslint-to-make-sure-your-code-respects-coding-standards

It will allow you to catch linting errors in your additions, such as spacing problems or missing translator comments in this case.